openmc/deplete/atom_number.py

```diff
@@ -2,73 +2,61 @@
 
 An ndarray to store atom densities with string, integer, or slice indexing.
 """
+from collections import OrderedDict
 
 import numpy as np
 
 
 class AtomNumber(object):
-    """AtomNumber module.
-
-    An ndarray to store atom densities with string, integer, or slice indexing.
+    """Stores local material compositions (atoms of each nuclide).
 
     Parameters
     ----------
-    mat_to_ind : OrderedDict of str to int
-        A dictionary mapping material ID as string to index.
-    nuc_to_ind : OrderedDict of str to int
-        A dictionary mapping nuclide name as string to index.
-    volume : OrderedDict of int to float
-        Volume of geometry.
-    n_mat_burn : int
-        Number of materials to be burned.
+    local_mats : list of str
+        Material IDs
+    nuclides : list of str
+        Nuclides to be tracked
+    volume : dict
+        Volume of each material in [cm^3]
     n_nuc_burn : int
         Number of nuclides to be burned.
 
     Attributes
     ----------
-    mat_to_ind : OrderedDict of str to int
-        A dictionary mapping cell ID as string to index.
-    nuc_to_ind : OrderedDict of str to int
-        A dictionary mapping nuclide name as string to index.
-    volume : numpy.array
-        Volume of geometry indexed by mat_to_ind.  If a volume is not found,
-        it defaults to 1 so that reading density still works correctly.
-    n_mat_burn : int
-        Number of materials to be burned.
+    index_mat : dict
+        A dictionary mapping material ID as string to index.
+    index_nuc : dict
+        A dictionary mapping nuclide name to index.
+    volume : numpy.ndarray
+        Volume of each material in [cm^3]. If a volume is not found, it defaults
+        to 1 so that reading density still works correctly.
+    number : numpy.ndarray
+        Array storing total atoms for each material/nuclide
+    materials : list of str
+        Material IDs as strings
+    nuclides : list of str
+        All nuclide names
+    burnable_nuclides : list of str
+        Burnable nuclides names. Used for sorting the simulation.
     n_nuc_burn : int
-        Number of nuclides to be burned.
-    n_mat : int
-        Number of materials.
+        Number of burnable nuclides.
     n_nuc : int
-        Number of nucs.
-    number : numpy.array
-        Array storing total atoms indexed by the above dictionaries.
-    burn_nuc_list : list of str
-        A list of all nuclide material names. Used for sorting the simulation.
-    burn_mat_list : list of str
-        A list of all burning material names. Used for sorting the simulation.
-    """
-
-    def __init__(self, mat_to_ind, nuc_to_ind, volume, n_mat_burn, n_nuc_burn):
+        Number of nuclidess.
```

Comment:
Spelling error.

(I'm not particularly looking for them, I just noticed this one)